caffe2/python/predictor/predictor_exporter_test.py

```diff
@@ -29,6 +29,23 @@
 import caffe2.python.predictor.predictor_py_utils as pred_utils
 from caffe2.proto import caffe2_pb2
 
+
+class MetaNetDefTest(unittest.TestCase):
+    def test_minimal(self):
+        '''
+        Tests that a NetsMap message can be created with a NetDef message
+        '''
+        # This calls the constructor for a caffe2_pb2.NetsMap
+        caffe2_pb2.NetsMap(key="test_key", value=caffe2_pb2.NetDef())
+
+    def test_adding_net(self):
+        '''
+        Tests that NetDefs can be added to MetaNetDefs
+        '''
+        meta_net_def = caffe2_pb2.MetaNetDef()
+        net_def = caffe2_pb2.NetDef()
+        meta_net_def.nets.add(key="test_key", value=net_def)
+
```

Comment:
These tests both fail when the messages in metanet_pb2 are in their own file that imports caffe2_pb2